index.html

```diff
@@ -9,6 +9,35 @@
 </head>
 
 <body>
+    <header>
+        <nav class="navegation">
+            <div class="logo">
+                <img src="images/logo 1.svg" alt="logo">
```

Comment:
Adicionaria um link pra a `home` nessa imagem.

```suggestion
                <a href=""><img src="images/logo 1.svg" alt="logo"></a>
```